src/InsertionsClient/Api/DefaultConfigUpdater.cs

```diff
@@ -209,7 +217,20 @@ public FileSaveResult[] Save()
 
             return results;
         }
-    
+
+        /// <summary>
+        /// Adds a blank line at the end of default.config, if needed.
+        /// </summary>
+        /// <param name="filePath">full path to default.config.</param>
+        /// <returns>true if the last characters are carriage return &amp; line feed.</returns>
+        /// <remarks>Calling <see cref="Object.ToString"/> on <see cref="XDocument"/> eliminates blank lines.</remarks>
+        internal static bool CheckIfLastFileLineIsBlank(string filePath)
+        {
+            byte[] bytes = File.ReadAllBytes(filePath);
+            return bytes[^2] == 13 && bytes[^1] == 10;
```

Comment:
Wouldn't checking only `bytes[^1] == 10` be sufficient? It would also be unix compatible.

I realized we use `Environment.NewLine` everywhere in the project. That means if we run the tool both on windows and linux, results will be different since they are using different line endings. Which will show up as lots of "changes" in the commit. Should we address this, maybe sometime in the future?

Reply:
I can test for Windows, test for 13 & 10, otherwise 10

- `Environment.NewLine` does the right thing: https://docs.microsoft.com/en-us/dotnet/api/system.environment.newline?view=netframework-4.8

Reply:
@bozturkMSFT , there is now a platform specific blank line test